### DISCUSSION TITLE
JupyterLab hangs indefinitely on terminal

### DISCUSSION THREAD

**Discussion by mengeziml** [jupyterlab/jupyterlab#15468] [General]
JupyterLab seems to be experiencing a bug with it's terminal interface.

I created a JupyterLab notebook instance on GCP in us-central1. Jupyter launches successfully and seems to work fine, but when I launch a terminal it just hangs on a blank screen with a flickering cursor. I waited over 30 mins for it. 

I created a new instance and the same issue persists.

I have logged the same issue with Google as well, but thought I should log it here as well incase it isn't a google issue

This what my JupyterLab instance looks like
![image](https://github.com/jupyterlab/jupyterlab/assets/48696302/cd3e1333-5f40-496f-aa06-4834aa27183d)

**Comment by krassowski** [comment]
@mengeziml can you please:
- try using another browser and report back if it is also a problem
- open a new bug report (rather than a discussion): https://github.com/jupyterlab/jupyterlab/issues/new/choose with details about version of the browser, OS, and jupyterlab
- provide a screenshot in the issue
- clarify if the other parts of the JupyterLab are responsible and the problem is limited to the terminal, or freezing the entire JupyterLab

Thanks!

**Reply by krassowski** [reply]
From the screenshot that you added in an edit after I written the message, it looks like the problem may be with your terminal configuration on jupyter-server side rather than with JupyterLab - but it cannot be said from the information here alone.

**Reply by mengeziml** [reply]
How do I address that?

Also I just tried doing the same on Edge. Problem still persists
![image](https://github.com/jupyterlab/jupyterlab/assets/48696302/9d1fb156-5d13-4d51-bfaa-425a40d5e027)

**Reply by mengeziml** [reply]
And sorry man. I've never had to log an issue on Github before. From the options I was seeing on the right hand side of my screen, discussion seemed to be the most appropriate one at the time

**Comment by ankitarane24** [comment]
Hi @mengeziml, 
Curious to know if you were able to solve this issue.
